quickjs.c

```diff
@@ -33922,13 +33968,23 @@ static JSValue JS_ReadFunctionTag(BCReaderState *s)
             vd->is_lexical = bc_get_flags(v8, &idx, 1);
             vd->is_captured = bc_get_flags(v8, &idx, 1);
 #ifdef DUMP_READ_OBJECT
-            bc_read_trace(s, "name: "); print_atom(s->ctx, vd->var_name); printf("\n");
+            if ((s->ctx->rt->dump_flags & DUMP_READ_OBJECT_FLAG) == DUMP_READ_OBJECT_FLAG) {
+                bc_read_trace(s, "%3d  %d%c%c%c %4d  ",
```

Comment:
Nice!

Reply:
I shall further simplify this in the next phase where `bc_read_trace` can output atoms directly.